test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -126,8 +126,10 @@ void TestSimpleCase()
             catch (InvalidOperationException) { }
         }
 
-        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 fails with "An attempt was made to load a program with an incorrect format."
-        void TestOldSavingAndLoading()
+        [ConditionalTheory(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 fails with "An attempt was made to load a program with an incorrect format."
+        [InlineData(null, false)]
```

Comment:
>InlineData [](start = 9, length = 10)

haven't seem this before. What is it for?

Reply:
The InlineData are values passed into the method (so gpuDeviceId=Null, and fallBackToCpu=false for the first run)

Each InlineData creates 2 extra tests.

This only works with Theory methods (not Fact methods)

---
In reply to: [245449923](https://github.com/dotnet/machinelearning/pull/2029#discussion_r245449923) [](ancestors = 245449923)